backend/cmd/start.go

```diff
@@ -242,19 +286,106 @@ func newStartCommand() *cobra.Command {
-	cmd.Flags().String(flagStoreInitialCluster, viper.GetString(flagStoreInitialCluster), "store initial cluster")
-	cmd.Flags().String(flagStoreInitialAdvertisePeerURL, viper.GetString(flagStoreInitialAdvertisePeerURL), "store initial advertise peer URL")
-	cmd.Flags().String(flagStoreInitialClusterState, viper.GetString(flagStoreInitialClusterState), "store initial cluster state")
-	cmd.Flags().String(flagStoreInitialClusterToken, viper.GetString(flagStoreInitialClusterToken), "store initial cluster token")
-	cmd.Flags().String(flagStoreNodeName, viper.GetString(flagStoreNodeName), "store cluster member node name")
+	cmd.Flags().String(flagEtcdClientURL, viper.GetString(flagEtcdClientURL), "list of URLs to listen on for client traffic")
+	_ = cmd.Flags().SetAnnotation(flagEtcdClientURL, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdPeerURL, viper.GetString(flagEtcdPeerURL), "list of URLs to listen on for peer traffic")
+	_ = cmd.Flags().SetAnnotation(flagEtcdPeerURL, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdInitialCluster, viper.GetString(flagEtcdInitialCluster), "initial cluster configuration for bootstrapping")
+	_ = cmd.Flags().SetAnnotation(flagEtcdInitialCluster, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdInitialAdvertisePeerURL, viper.GetString(flagEtcdInitialAdvertisePeerURL), "list of this member's peer URLs to advertise to the rest of the cluster")
+	_ = cmd.Flags().SetAnnotation(flagEtcdInitialAdvertisePeerURL, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdInitialClusterState, viper.GetString(flagEtcdInitialClusterState), "initial cluster state (\"new\" or \"existing\")")
+	_ = cmd.Flags().SetAnnotation(flagEtcdInitialClusterState, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdInitialClusterToken, viper.GetString(flagEtcdInitialClusterToken), "initial cluster token for the etcd cluster during bootstrap")
+	_ = cmd.Flags().SetAnnotation(flagEtcdInitialClusterToken, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdNodeName, viper.GetString(flagEtcdNodeName), "human-readable name for this member")
+	_ = cmd.Flags().SetAnnotation(flagEtcdNodeName, "categories", []string{"store"})
 	cmd.Flags().Bool(flagNoEmbedEtcd, viper.GetBool(flagNoEmbedEtcd), "don't embed etcd, use external etcd instead")
+	_ = cmd.Flags().SetAnnotation(flagNoEmbedEtcd, "categories", []string{"store"})
+
+	// Etcd TLS flags
+	cmd.Flags().String(flagEtcdCertFile, viper.GetString(flagEtcdCertFile), "path to the client server TLS cert file")
+	_ = cmd.Flags().SetAnnotation(flagEtcdCertFile, "categories", []string{"store"})
+	cmd.Flags().String(flagEtcdKeyFile, viper.GetString(flagEtcdKeyFile), "path to the client server TLS key file")
+	_ = cmd.Flags().SetAnnotation(flagEtcdKeyFile, "categories", []string{"store"})
+	cmd.Flags().Bool(flagEtcdClientCertAuth, viper.GetBool(flagEtcdClientCertAuth), "enable client cert authentication")
```

Comment:
What's the default value for `etcd-client-cert-auth`?

Reply:
It's a boolean value, so the default value is `false`, which makes sense.